### PR TITLE
Update README with project maintenance status

### LINKED ISSUE #292: Community chat?
I'm interested in contributing, but I'm not super technically skilled and would be interested to ask a few questions... did anyone think of setting up a Matrix room or Gitter? Would be helpful for someone like me!

### PR DESCRIPTION
**Pull request by DougInAMug**
Update README.md to reflect the changes to the project status, and link the matrix Room

- closes #292

### CODE REVIEW

**Comment by shinenelson** [comment]
2 suggestions : 
1. We can add a [matrix badge](https://github.com/matrix-org/matrix.to#how-can-i-put-a-badge-on-my-website-linking-to-my-matrix-room) to the set of other badges on the readme which would make the link to the room in a more prominent place.
2. Rather than adding numbered links to the end of a line, can we probably link words? Or alternatively, make them footnotes [^1] [^2] which ironically is [not supported] by GitHub-flavored Markdown ( GFM ) which renders markdown in places like this comment; but I think it works in rendering readmes because it uses a different markdown renderer which [does support footnotes].

[^1]: https://www.markdownguide.org/extended-syntax#footnotes
[^2]: https://kramdown.gettalong.org/syntax.html#footnotes
[not supported]: https://github.com/github/markup/issues/498
[does support footnotes]: https://github.com/github/cmark-gfm/pull/64

**Comment by DougInAMug** [comment]
I like all those suggestions @shinenelson - I would wait for the badge decision (pending on history decision) before merging :)

**Comment by shinenelson** [comment]
Would it make sense to use a heading like **Project Status** or **Project Maintenance Status Update** or something along those lines? Timestamps can quickly get outdated.

Also, it would nice to link to #235 and #292 to indicate where we started.

**Comment by DougInAMug** [comment]
Thanks for suggestions again @shinenelson. I mentioned the issues you linked, and rewrote the introduction to avoid timestamp, and included matrix badge :)

File: README.md
Comment (shinenelson) [inline README.md:119]:
```suggestion
[jsdeliver-badge]: https:/img.sheilds.io/jsdelivr/npm/hm/fork-awesome "JSDeliver badge"
```

For the sake of consistency, since we use shields.io everywhere else.

( PS : I personally prefer flat badges )

File: README.md
Comment (shinenelson) [inline README.md:114]: We have 126 contributors now. In my opinion, this badge is unmaintainable since it is not a dynamic badge. Unless the all-contributors bot or the all contributors project provides this, this badge will be outdated most of the time.

File: README.md
Comment (shinenelson) [inline README.md:123]:
```suggestion
[chat-badge]: https://img.shields.io/matrix/forkawesome:matrix.org?label=%23forkawesome%3Amatrix.org "chat (matrix) badge"
```

makes the badge accessible.

File: README.md
Comment (shinenelson) [inline README.md:124]:
```suggestion
[chat-link]: https://matrix.to/#/#forkawesome:matrix.org
```
gives users the choice to choose their preferred Matrix client and not force everyone to use Element.

File: README.md
Comment (shinenelson) [inline README.md:10]: @xuv@merveilles.town is not a valid URL. Perhaps you meant

```suggestion
With gratitude to Dave Gandy and the Font Awesome team, [Julien](https://github.com/xuv) [Deswaef](https://merveilles.town/@xuv) forked Font Awesome 4.7 into Fork Awesome 1.0 in [Feb. 2018](https://github.com/ForkAwesome/Fork-Awesome/releases/tag/1.0.0), to continue building on the amazing resource in a fully reproduceable way, as a distributed community of contributors.
```

Some more updates to the content

```suggestion
With gratitude to Dave Gandy and the Font Awesome team, [Julien](https://github.com/xuv) [Deswaef](@xuv@merveilles.town) forked Font Awesome 4.7 into [Fork Awesome 1.0](https://github.com/ForkAwesome/Fork-Awesome/releases/tag/1.0.0) in February 2018, to continue building on the amazing resource in a truly open source fashion; with the distributed community of contributors and for the community.
```

File: README.md
Comment (shinenelson) [inline README.md:12]: - Perhaps, there needs to be a line break before 🌱 ? :thinking: 
- _faithfully maintaining the project **for a year**_
- hadn't => did not
- There are too many sentence joins with commas
- PR's => pull requests
- The matrix link is still here

File: README.md
Comment (shinenelson) [inline README.md:8]: Since we are revamping this section, can we also move the link pointers to the bottom of the section? Like it is done for the badges. That way the content becomes readable and _un-cluttered_ from links sprinkled over the place.

**Review by shinenelson — CHANGES_REQUESTED** [review]
Nice revamp of the introductory section. I suggested a few more changes that could make the changes even better.

File: README.md
Comment (DougInAMug) [reply README.md:119]: I think that should be img.sh**ie**lds.io... Otherwise cool.

I slightly prefer rounded, but I don't mind, just want them all the same style

File: README.md
Comment (DougInAMug) [reply README.md:114]: Oh true! Hmm... I think it's nice to keep, as a show of appreciation. Perhaps we can find some way to remember every time before we make a release? 'Release checklist', or something like that.

File: README.md
Comment (DougInAMug) [reply README.md:123]: Maybe you're right, that everyone knows about matrix by now? I just thought my original formulation would be clearer for anyone who hadn't heard of matrix by now.

File: README.md
Comment (shinenelson) [reply README.md:119]: The feeling when you realize that you messed up a suggestion for a fix :facepalm:

fixed the suggestion :see_no_evil:

File: README.md
Comment (shinenelson) [reply README.md:114]: That is a good idea. I agree.

For the immediate release ( 1.2 ), there was only one contributor that was added and who did everything that went into the release. We should definitely acknowledge them.

And also, since we are already updating this line, it would make sense to send that update with this change-set.

However, we definitely need to _create_ that Release checklist :smile:

File: README.md
Comment (shinenelson) [reply README.md:123]: _matrix_ is still retained. and the `#<channel-name>` is mainstream now ( thanks to Slack ). people ( even the ones who has not heard of matrix before ) would at least read "_channel forkawesome on service called matrix.org_"

File: README.md
Comment (DougInAMug) [reply README.md:12]: - Don't you you mean, after :seedling: ?
- I think we can leave contribution time ambiguous... he was contributing longer than a year
- 'Did not' seems clearer :+1: 
- I do like commas! But perhaps you're right. I can try and split it up into more sentences. Like this ;p
- PR -> pull request :+1: 
- Yes, link still there intentionally. I think it's good to have that quite present, especially after I talked to some others.

File: README.md
Comment (DougInAMug) [reply README.md:8]: Not against that.

File: README.md
Comment (shinenelson) [reply README.md:12]: I do not feel too strongly about the matrix link, but if you are going to keep it, please change the URL to use matrix.to.

File: README.md
Comment (DougInAMug) [reply README.md:8]: Actually, I've had a change of thought on this :sweat_smile: I agree it disturbs the flow of reading raw a little bit, but having to maintain currency with links stored elsewhere is also irritating. I could see it making more sens if we were referring to the same URL multiple times, but al our URL's are unique...

I would even suggest moving the badge URLs to the badges, but happy to leave.

Comment (shinenelson) [reply README.md:8]: My suggestion was not to move the links all the way to the bottom of the page, but to the bottom of the section ( that is, right above the **How to use** heading ). The links would still be in view.

If you feel strongly about not moving the links, then okay. But let us not move the badge links.

**Review by shinenelson — APPROVED** [review]

File: README.md
Comment (shinenelson) [reply README.md]: This suggestion was messed up from the start. The fix to the first mess up still did not fix it. There was a `/` missing after the `https:/` :facepalm:

**Pull request merged**
